tests/travis/run_test.sh

```diff
@@ -61,9 +61,13 @@ if [ ${TASK} == "r_test" ]; then
 
     set -e
     export _R_CHECK_TIMINGS_=0
-    wget https://cran.rstudio.com/bin/macosx/R-latest.pkg  -O /tmp/R-latest.pkg
-    sudo installer -pkg "/tmp/R-latest.pkg" -target /
-    Rscript -e "install.packages('devtools', repo = 'https://cran.rstudio.com')"
+    
+    if [[ ${TRAVIS_OS_NAME} == "osx" ]]; then
```

Comment:
Line 66 wget's a pkg for OSX, line 67 calls "installer" which seems to be an OSX tool? Has this test been running for linux at all?